crates/db/src/schema.rs

```diff
@@ -3,7 +3,7 @@
 /// Column families alias type
 pub type Col = u8;
 /// Total column number
-pub const COLUMNS: u32 = 37;
+pub const COLUMNS: u32 = 38;
```

Comment:
We should define a migration when adding a new column. If we cannot do an actual migration yet, we can check that the db is empty.